### PR TITLE
Docs v3

### PR DESCRIPTION
**Pull request by alexand3rwilke**
## Summary
- light docs page for the new website
- overview of key parts
- added xbi docs from xbi repo
- added collator part from previous docs

### CODE REVIEW

File: docs/main/docs/collator/mainnet/collator-requirements-mainnet.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+- Collator Bond: 10 T0RN Tokens ([1 drip from testnet faucet](https://dev.net.t3rn.io/faucet/))
+
+## Hardware Requirements
+| Component | Requirement |
+|---|---|
+| System | Ubuntu 22.04 |
+| CPU | 4 cores |
+| Memory | 16 GB |
+| Hard Disk | 100 GB SSD Minimum |
```
Comment (chiefbiiko) [inline docs/main/docs/collator/mainnet/collator-requirements-mainnet.md:15]:
```suggestion
| Storage | min. 100 GB SSD |
```

File: docs/main/docs/collator/mainnet/mainnet-collator.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
+# Run a t3rn mainnet Collator
+
```
Comment (chiefbiiko) [inline docs/main/docs/collator/mainnet/mainnet-collator.md:2]: maybe we wanna add a visible NOTE here that this is to be rolled out

File: docs/main/docs/collator/testnet/testnet-collator.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
+---
+sidebar_position: 1
+---
+
+# Run a t0rn Testnet Collator
+
```
Comment (chiefbiiko) [inline docs/main/docs/collator/testnet/testnet-collator.md:6]: similar NOTE here that we will be updating our bootnode and release version once we have revamped the testnet

**Review by chiefbiiko — COMMENTED** [review]
Let's try to use one case format for all "side effect" occurrences.

"Side Effect" ?

File: docs/main/docs/about_t3rn/what-is-t3rn.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+<p align="center">
+    <img height="150" src="/img/t3rn-overview.png?raw=true"/>
+</p>
+
+
+# Interoperability - How does t3rn handle multichain transaction?
+When a multichain smart contract is executed, it creates transactions on other blockchains, called [Sideeffects](components/sfx/sfx-overview).
+
+Side Effects are operations that contain parameters for a transaction on a target blockchain. The requester of a side effect specifies the fee he wants to pay for the execution and submits it to the Circuit.
```
Comment (petscheit) [inline docs/main/docs/about_t3rn/what-is-t3rn.md:24]: we have renamed fee to reward, as it makes more semantical sense

File: docs/main/docs/components/sfx-overview.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+```
+
+#### target:
+`target` describes the destination consensus system the Sideeffect should be executed on. 
+
+#### max_reward 
+`max_reward` sets the reward for the Executor in `TRN`.
+
+#### insurance
+`insurance` sets the minimum amount of insurance and Executor has to Bond. This is important for optimistic transactions.
```
Comment (petscheit) [inline docs/main/docs/components/sfx-overview.md:30]: insurance is just as important in escrow. It incentives executors to confirm the transactions they have one the bidding on

File: docs/main/docs/components/sfx-overview.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+#### encoded_action
+`encoded_action` sets the action that should be done on the target chain e.g.: call `function x` on `contract y`.
```
Comment (petscheit) [inline docs/main/docs/components/sfx-overview.md:33]: correct, but encoded action is an ID, so for transfers we use tran

**Review by petscheit — APPROVED** [review]
Looks good!! Nice stuff mate!

File: docs/main/docs/about_t3rn/token-economics/tokenomics-inflation.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+|12|0.2|0.5|0.8|1.1|4.40 %|
+|24|0.2|0.5|0.8|1.1|4.40 %|
+|48|0.2|0.5|0.8|1.1|4.40 %|
+|72|0.2|0.5|0.8|1.1|4.40 %|
+
+## Gas Fees
+
+| | Collators | Executors | Attesters | Authors | Treasury | Users |
+|---|---|---|---|---|---|---|
+| Any Other TX| 99% | N/A | N/A | N/A | 1% | 0% |
```
Comment (MaciejBaj) [inline docs/main/docs/about_t3rn/token-economics/tokenomics-inflation.md:30]:
```suggestion
| Any TX| 80% | N/A | N/A | N/A | 20% | 0% |
```

File: docs/main/docs/about_t3rn/token-economics/tokenomics-inflation.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+| Contracts Call TX | 19% | N/A | N/A | 80% | 1% | 0% |
```
Comment (MaciejBaj) [inline docs/main/docs/about_t3rn/token-economics/tokenomics-inflation.md:31]:
```suggestion
| Contracts Call TX | 10% | N/A | N/A | 80% | 10% | 0% |
```

File: docs/main/docs/about_t3rn/token-economics/tokenomics-inflation.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+| SFX + XTX Confirm Tx | 19% | 80% | N/A | N/A | 1% | 0% |
```
Comment (MaciejBaj) [inline docs/main/docs/about_t3rn/token-economics/tokenomics-inflation.md:32]:
```suggestion
```

File: docs/main/docs/about_t3rn/token-economics/tokenomics-inflation.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+| SFX Attestations | 19% | N/A | N/A | N/A | 1% | 0% |
```
Comment (MaciejBaj) [inline docs/main/docs/about_t3rn/token-economics/tokenomics-inflation.md:33]:
```suggestion
```

File: docs/main/docs/about_t3rn/token-economics/tokenomics-inflation.md [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+# Tokenomics and Inflation
+
+**Total token supply**: `100,000,000`
+
+**Token ticker**: `$TRN`
+
+**Inflation**: `4.40%` (yearly)
+
+
+t3rn network participants are able to get `remuniated` through Inflation and Gas fees:
```
Comment (MaciejBaj) [inline docs/main/docs/about_t3rn/token-economics/tokenomics-inflation.md:14]:
```suggestion
t3rn network participants are able to get `remunerated ` through Inflation and Gas fees:
```

**Comment by alexand3rwilke** [comment]
Duplicate #519

**Pull request closed without merging**